resource/src/routes/route.mustache

```diff
@@ -1,26 +1,33 @@
+{{#isUser}}
+const { Router } = require('express');
+{{/isUser}}
 const { createRouter } = require('../utils/router');
 const {{camelCasePlural}} = require('../controllers/{{snakeCasePlural}}');
 
+{{#isUser}}
 const serialize = {{camelCase}} => {
   const data = Object.assign({}, {{camelCase}});
-  {{#isUser}}
   delete data.password;
-  {{/isUser}}
   return data;
 };
 
-{{#isUser}}
-const template = {
-  post: {
-    '/signin': function *(req, res) {
-      const { user, token } = yield {{camelCasePlural}}.signin(req.body.email, req.body.password);
-      res.json({ token, user: serialize(user) });
-    },
-    '/signout': function *(req, res) {
-      res.json(yield {{camelCasePlural}}.signout(req.body));
-    }
-  }
-};
+const customRouter = new Router()
+  .post('/signin', function *(req, res) {
+    const { user, token } = yield {{camelCasePlural}}.signin(req.body.email, req.body.password);
+    res.json({ token, user: serialize(user) });
+  })
+  .post('/signout', function *(req, res) {
+    res.json(yield {{camelCasePlural}}.signout(req.body));
+  });
+
+const defaultRouter = createRouter({{camelCasePlural}}, serialize);
+customRouter.socket = defaultRouter.socket;
+
+customRouter.use(defaultRouter);
+
+module.exports = customRouter;
```

Comment:
Yeah, you're right. There's nothing gained here by nesting the routers rather than just assigning directly. I originally did it because I had a memory of a custom `.get` function that removed the `.password` field, but that's done through a serialiser now and that fact obviously just hadn't settled in my brain.

I am inclined to leave it there for no other reason than it's a way to capture the pattern for doing this, and we don't really have anywhere else to put it right now. Maybe once we document things in earnest we can simplify this default path and move the pattern out into docs as a howto.